### ISSUE TITLE
Document tooltip/popover methods

### ISSUE THREAD

**Issue by Johann-S** [twbs/bootstrap#22188]
Due to my bad english I prefer to open an issue. For tooltip and popover we have a lack of documentation for ``enable()``, ``disable()`` and ``toggleEnabled()``

X-Ref : https://github.com/twbs/bootstrap/issues/22162#issuecomment-286187379

For example (but maybe it's to light)  : 

#### `.tooltip('enable')`

Enable an element's tooltip.

```javascript 
$('#element').tooltip('enable')
```

#### `.tooltip('disable')`

Disable an element's tooltip.

```javascript 
$('#element').tooltip('disable')
```

#### `.tooltip('toggleEnabled')`

Allow to enable or disable an element's tooltip.

```javascript 
$('#element').tooltip('toggleEnabled')
```

**Comment by sidp0** [comment]
This seems like a problem I might be able to tackle as a first contribution. Would that be okay?

**Comment by Johann-S** [comment]
Sure @sidp0 go on 👍

**Comment by Johann-S** [comment]
Closed by #22246
